src/Handles/Handle.tsx

```diff
@@ -162,7 +165,15 @@ const Handle = React.forwardRef<HTMLDivElement, HandleProps>((props, ref) => {
       aria-valuetext={getIndex(ariaValueTextFormatterForHandle, valueIndex)?.(value)}
       aria-orientation={direction === 'ltr' || direction === 'rtl' ? 'horizontal' : 'vertical'}
       {...restProps}
-    />
+    >
+      {customHandle && (
```

Comment:
Handle accepts the `render` prop function, which seems to be able to do the same thing. Are they redundant to each other?

Reply:
It is possible, seems easiest to clone the element and add custom components as children. Although the styles of the handleNode will still apply to children so have to either manually set or set visiblity.

```
const customRender = (handle: React.ReactElement) => {
  return React.cloneElement(handle, {
    style: {
      ...handle.props.style,
      visibility: 'hidden',
    }
  }, 
    <CustomHandle /> // Add style 'visiblity': 'visible' to child component
  );
};
```
And simply passing in 
`handleRender={customRender}`

Happy to close the PR

Reply:
Ok, let's close it.

Reply:
Just ran into this and am a little surprised that this seems to be the supposed way to customize the slider handle.

The [docs on React.cloneElement](https://react.dev/reference/react/cloneElement) state: "Pitfall: Using cloneElement is uncommon and can lead to fragile code." So I would favor to have a more straight-forward solution to custom handle elements. Otherwise, it would be nice to have at least some documentation and examples. But seems like the README is not up to date to the v10 changes.